packages/file_selector/file_selector_android/android/src/main/java/io/flutter/plugins/file_selector/FileSelectorDelegate.java

```diff
@@ -73,14 +76,26 @@ public FileSelectorDelegate(final Activity activity) {
 
   @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
   public void getDirectoryPath(MethodCall methodCall, MethodChannel.Result result) {
-    if (!setPendingMethodCallAndResult(methodCall, result)) {
+    if (setPendingMethodCallAndResult(methodCall, result)) {
```

Comment:
It seems to me that I've seen this `if` using distinct conditions.

Reply:
oh that was changed because android studio said so, if you check the setPendingMethodCallAndResult, it changed the true for false for clearness.

Reply:
Please let @VanesaOshiro know.